ci_framework/roles/edpm_build_images/tasks/package.yml

```diff
@@ -24,3 +24,12 @@
       buildah bud -f ./Containerfile.ramdisk
       -t ironic-python-agent:{{ cifmw_edpm_build_images_tag }}
       --logfile {{ cifmw_edpm_build_images_basedir }}/logs/edpm_images/ironic_python_agent_container_package.log
+
+- name: Dump edpm container images in the file
+  args:
+    chdir: "{{ cifmw_edpm_image_builder_repo_path }}"
+  ansible.builtin.shell: |
+    sudo buildah images | grep "{{ item }}" | tee {{ cifmw_edpm_build_images_basedir }}/logs/containers-built.log
```

Comment:
you probably want `tee -a` else it will override upon each iteration.
We may probably avoid iteration using a tweaked `| grep` such as `| grep -E '(edpm-hardened-uefi|ironic-python-agent)'` (NOT TESTED).